### PR TITLE
:sparkles: Added an imager welcome example

### PR DESCRIPTION
**Pull request by Arthurdw**
A new example which shows how you would make a member welcomer, this example also shows a way on how you can import all cogs from a directory. 
image

Variant:
image

### Changes

-   `adds`: A welcome example

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My code follows the pep8 code style.

### CODE REVIEW

File: examples/discord-welcomer/cogs/welcome.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    BASE_API_URL = "https://api.xiler.net"
+    REQUEST_HEADERS = {
+        "Content-Type": "application/json",
+    }
+
+    def __init__(self, client: Client):
+        super().__init__(client)
+        self.__session: ClientSession = ClientSession(self.BASE_API_URL)
+
+        with open("./cogs/templates/welcome.html", "r") as f:
```
Comment (Sigmanificient) [inline examples/discord-welcomer/cogs/welcome.py:33]:
```suggestion
        with open("./cogs/templates/welcome.html") as f:
```

Comment (Sigmanificient) [inline examples/discord-welcomer/cogs/welcome.py:33]:
```suggestion
        with open("./cogs/templates/welcome.html") as f:
```

File: examples/discord-welcomer/main.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    continue
+
+                if obj := getattr(module, definition, None):
+                    parents = getattr(obj, "__bases__", None) or []
+
+                    if Cog in parents:
+                        self.load_cog(obj)
+                        self.loaded_cogs += 1
+                        logging.info(
+                            "Successfully loaded %s (%s)" % (definition, container))
```
Comment (Sigmanificient) [inline examples/discord-welcomer/main.py:57]:
```suggestion
                            "Successfully loaded %s (%s)" % (definition, container)
                        )
```

File: examples/discord-welcomer/cogs/templates/welcome.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            width: 200px;
+            height: 200px;
+
+            border-radius: 50%;
+        }
+
+        .img-wrapper {
+            margin-left: 70px;
+
+            border: 4px solid #ffd53d;
```
Comment (Sigmanificient) [inline examples/discord-welcomer/cogs/templates/welcome.html:44]:
```suggestion
            border: 4px solid white;
```

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: examples/discord-welcomer/cogs/templates/welcome.html (hunk 3 above)
Comment (Arthurdw) [reply examples/discord-welcomer/cogs/templates/welcome.html:44]: Why?

Comment (Sigmanificient) [reply examples/discord-welcomer/cogs/templates/welcome.html:44]: match the white title color

Comment (Arthurdw) [reply examples/discord-welcomer/cogs/templates/welcome.html:44]: Pincer theme tho :/

Comment (Arthurdw) [reply examples/discord-welcomer/cogs/templates/welcome.html:44]: Pincer theme tho :/

**Review by Sigmanificient — APPROVED** [review]

**Pull request merged**